lib/hexpm_web/templates/dashboard/organization/index.html.eex

```diff
@@ -5,6 +5,84 @@
     <%= render DashboardView, "_sidebar.html", assigns %>
   </div>
 
+  <div class="col-sm-9 email">
+    <div class="panel panel-default">
+      <div class="panel-heading">Emails</div>
+      <div class="panel-body">
+        <p>
+          The <strong>primary</strong> email address will be used when Hex.pm communicates with you.
+          The <strong>public</strong> email address will be displayed on your profile page.
+        </p>
+      </div>
+      <ul class="list-group">
+        <%= for email <- @emails do %>
+          <li class="list-group-item clearfix">
+            <span class="address"><%= email.email %></span>
+
+            <%= if email.primary do %>
+              <span class="label label-success">Primary</span>
+            <% end %>
+            <%= if email.public do %>
+              <span class="label label-success">Public</span>
+            <% end %>
+            <%= if email.gravatar do %>
+              <span class="label label-success">Gravatar</span>
+            <% end %>
+
+            <%= form_tag(Routes.organization_path(Endpoint, :delete_email, @organization.name), method: :delete, class: "action") do %>
+              <input type="hidden" name="email" value="<%= email.email %>">
+              <button type="submit" class="btn btn-link btn-svg" <%= if email.primary do %>disabled<% end %>>
+                <%= HexpmWeb.ViewIcons.icon(:octicon, :trashcan) %>
+              </button>
+            <% end %>
+
+            <%= if email.verified and not email.primary do %>
```

Comment:
Do we need to allow organizations to change their primary email?

I think it makes sense for organizations to set gravatar and public emails.
But for primary email, I'm not so sure.